### PR TITLE
POC: amountSpecified matches deltas

### PR DESCRIPTION
**Pull request by ewilz**
## Related Issue
Which issue does this pull request resolve?

## Description of changes

### CODE REVIEW

File: test/PoolManager.t.sol [hunk 1]
```diff
@@ -464,7 +464,7 @@ contract PoolManagerTest is Test, Deployers, GasSnapshot {
         modifyLiquidityRouter.modifyLiquidity{value: 1 ether}(nativeKey, liqParams, ZERO_BYTES);
 
         IPoolManager.SwapParams memory params =
-            IPoolManager.SwapParams({zeroForOne: true, amountSpecified: 100, sqrtPriceLimitX96: SQRT_RATIO_1_2});
+            IPoolManager.SwapParams({zeroForOne: true, amountSpecified: -100, sqrtPriceLimitX96: SQRT_RATIO_1_2});
```
Comment (hensha256) [inline test/PoolManager.t.sol:467]: This identical struct is used all over this file, and DynamicFees.t.sol. I wonder if we should just save it as our default `SWAP_PARAMS` in `Deployers.sol` like we have `LIQ_PARAMS` etc in there. And then al these files can just use that constant defined in 1 place?

Comment (hensha256) [reply test/PoolManager.t.sol:467]: or `SWAP_EXACT_IN_PARAMS` or something

**Review by hensha256 — COMMENTED** [review]
Looking great, much simpler than i thought it was going to be lol

File: test/PoolManager.t.sol (hunk 1 above)
Comment (hensha256) [reply test/PoolManager.t.sol]: I added this to our #390

Comment (ewilz) [reply test/PoolManager.t.sol]: good call

**Pull request merged**